README: add mirror example and usage of -X (#68)
bwesterb/mtc: README.md

assertions. In the future this could for instance be used for
serialized DNSSEC proofs.

> We can also create an assertion request that roughly matches an existing X.509

Reviewer: Just to avoid having to clarify what 'roughly matches' means:
```suggestion
We can also create an assertion request derived from an existing X.509
```

authentication path
 8964f010faa9e499b21917f8792b541b7b1ac19f313a5d53094c698c2edc330b
```

### Mirroring a CA

We can set up a new mirror with the `mtc mirror new` command:

```
$ mtc mirror new ca.example.com/path
```

This will download the `ca-params`
from `https://ca.example.com/path/mtc/v04b/ca-params` and
set up a directory structure similar to that of a CA:

```
$ find .
.
./www
./www/mtc
./www/mtc/v04b
./www/mtc/v04b/ca-params
./www/mtc/v04b/batches
./tmp
```

> To bring the mirror up to date with the CA, use the `update` command:

Reviewer: We should add a server mode for the mirror too, so that it periodically updates from the CA and serves the files over HTTP.

Author: Indeed: #58
